aces/Loris
# [dqt] New ProgressBar/Remove Smarty/with needed fixes for performance

## Brief summary of changes

This PR contains the enhancements that had been done for the ABCD project. The smarty template had to be removed to add the ProgressBar and a few places in the DQT had to be modified to load without any sessions or visits data. Note this change helps when the couchdb has lots of sessions for the project.

#### Testing instructions (if applicable)

1. Checkout PR
2. Visit the DQT
3. See progressbar.
